addons/mail/static/src/messaging/entity/messaging/messaging.js

```diff
@@ -59,28 +59,35 @@ function MessagingFactory({ Entity }) {
          * @param {Object} param0
          * @param {integer} param0.id
          * @param {string} param0.model
+         * @param {Object} [param2]
+         * @param {boolean} [param2.forceOpenDocument]
          */
-        async redirect({ id, model }) {
-            if (model === 'mail.channel') {
+        async redirect({ id, model }, { forceOpenDocument = false}={}) {
```

Comment:
space operators and before }
```suggestion
        async redirect({ id, model }, { forceOpenDocument = false } = {}) {
```